lib/dialects/postgres/connection-manager.js

```diff
@@ -109,18 +110,21 @@ class ConnectionManager extends AbstractConnectionManager {
           return;
         }
         responded = true;
+        debug(`connection acquire`);
```

Comment:
similar thing happen in mysql no `uuid` when using transactions. Now trying pools, may be they have uuid for connection